lavalink/src/http.rs

```diff
@@ -283,6 +291,14 @@ pub fn get_route_planner(
 /// Unmark an IP address as being failed, meaning that it can be used again.
 ///
 /// The response will not include a body on success.
+///
+/// # Errors
+///
+/// See the documentation for [`http::Error`].
+///
+/// # Panics
+///
+/// Panics if serde serialization fails.
```

Comment:
This section can be removed, with the `unwrap()` replaced by `expect("valid json")` as it's statically known to be valid

Reply:
I'll make this change in a followup PR